velox/common/caching/SsdFile.cpp

```diff
@@ -965,6 +948,9 @@ void SsdFile::disableFileCow() {
   const std::unordered_map<std::string, std::string> attributes = {
       {std::string(LocalWriteFile::Attributes::kNoCow), "true"}};
   writeFile_->setAttributes(attributes);
+  if (evictLogWriteFile_ != nullptr) {
+    evictLogWriteFile_->setAttributes(attributes);
```

Comment:
We don't disable checkpoint file?

Reply:
It is disabled in a separate path as we haven't switched checkpoint files to velox fs.